OpenRA.Game/Game.cs

```diff
@@ -23,6 +23,7 @@
 using OpenRA.Network;
 using OpenRA.Primitives;
 using OpenRA.Support;
+using OpenRA.Traits;
```

Comment:
This using isn't needed anymore, so please remove it.

Reply:
removed